project/src/main/java/com/google/sps/data/PlacesFetcher.java

```diff
@@ -104,25 +107,40 @@ public PlacesFetcher(GeoApiContext geoApiContext) {
      *     for places or for places details
      */
     public ImmutableList<Place> fetch(UserPreferences preferences) throws FetcherException {
-        PlacesSearchResult[] placesSearchResult;
+        Map<String, Set<String>> placesSearchResults = new HashMap<>();
```

Comment:
I changed to immutableSet builder
But I am having trouble doing that with the map because I need to have the ability to get objects from it in order to add a cuisine to a place already in the map. 
So is it fine to work with a regular map and then when passing it the the next method pass an immutable copy?